tasks/system_probe.py

```diff
@@ -669,6 +669,85 @@ def test(
         raise Exit(code=1, message="system-probe tests failed")
 
 
+@task(
+    help={
+        "package": "The package to test. REQUIRED ",
+        "skip_object_files": "Skip rebuilding the object files.",
+        "run": "The name of the test to run. REQUIRED",
+    }
+)
+def test_debug(
+    ctx,
+    package,
+    bundle_ebpf=False,
+    runtime_compiled=False,
+    co_re=False,
+    skip_linters=False,
+    skip_object_files=False,
+    run=None,
+    windows=is_windows,
+    failfast=False,
+    kernel_release=None,
+    timeout=None,
+):
+    """
+    Run delve on a specific system-probe test.
+    """
+    if not run:
+        raise Exit(
+            code=1, message="test name must be specified with the --run option")
```

Comment:
I think you can get the same behavior by not defining a default value for the `run` argument.

Reply:
Indeed, nice catch!